lib/structures/ForumChannel.js

```diff
@@ -0,0 +1,175 @@
+     * @arg {Array<String>} [options.message.stickerIDs] An array of IDs corresponding to stickers to send
+     * @arg {String} options.name The thread channel name
+     * @arg {Number} [options.rateLimitPerUser] The time in seconds a user has to wait before sending another message (0-21600) (does not affect bots or users with manageMessages/manageChannel permissions)
+     * @arg {String} [options.reason] The reason to be displayed in audit logs
+     * @arg {Object | Array<Object>} [file] A file object (or an Array of them)
+     * @arg {Buffer} file.file A buffer containing file data
+     * @arg {String} file.name What to name the file
+     * @returns {Promise<ThreadChannel>}
+     */
+    createThread(options, file) {
+        return this.client.createThread.call(this.client, this.id, options, file);
+    }
+
+    /**
+     * Create a channel webhook
+     * @arg {Object} options Webhook options
+     * @arg {String} [options.avatar] The default avatar as a base64 data URI. Note: base64 strings alone are not base64 data URI strings
+     * @arg {String} options.name The default name
+     * @arg {String} [reason] The reason to be displayed in audit logs
+     * @returns {Promise<Object>} Resolves with a webhook object
+     */
+    createWebhook(options, reason) {
+        return this.client.createChannelWebhook.call(this.client, this.id, options, reason);
+    }
+
+    /**
+     * Get all archived threads in this channel
+     * @arg {String} type The type of thread channel. Note: Only "public" is available in forum channels
+     * @arg {Object} [options] Additional options when requesting archived threads
+     * @arg {Date} [options.before] List of threads to return before the timestamp
+     * @arg {Number} [options.limit] Maximum number of threads to return
+     * @returns {Promise<Object>} An object containing an array of `threads`, an array of `members` and whether the response `hasMore` threads that could be returned in a subsequent call
+     */
+    getArchivedThreads(type, options) {
+        return this.client.getArchivedThreads.call(this.client, this.id, type, options);
+    }
+
+    /**
+     * Get all invites in the channel
+     * @returns {Promise<Array<Invite>>}
+     */
+    getInvites() {
+        return this.client.getChannelInvites.call(this.client, this.id);
+    }
+
+    /**
+     * Get all the webhooks in the channel
+     * @returns {Promise<Array<Object>>} Resolves with an array of webhook objects
+     */
+    getWebhooks() {
+        return this.client.getChannelWebhooks.call(this.client, this.id);
+    }
+
+    toJSON(props = []) {
+        return super.toJSON([
+            "availableTags",
+            "defaultAutoArchiveDuration",
+            "defaultForumLayout",
+            "defaultReactionEmoji",
+            "defaultSortOrder",
+            "defaultThreadRateLimitPerUser",
+            "flags",
+            "lastMessageID",
+            "nsfw",
+            "permissionOverwrites",
```

Comment:
Don't include this, incompatible with json

Reply:
Fixed all JSON shiz 👍🏻